test/libsolidity/syntaxTests/nameAndTypeResolution/265_new_for_non_array.sol

```diff
@@ -1,8 +1,7 @@
 contract C {
     function f(uint size) public {
-        var x = new uint(7);
+        uint[] storage x = new uint(7);
```

Comment:
The type would be `uint`, but perhaps just `new uint(7)` (without assignment) works fine, or does it generate "this statement does not have an effect" warning?

Reply:
it doesn't complain, it was a misinterpretation of `new uint(7)` on my side.